index.bs

```diff
@@ -3217,7 +3217,7 @@ Wide Review</h5>
 	is likely not to be considered as sufficient evidence
 	that the group has solicited wide review.
 
-	A [=Working Group=] could present evidence that wide review has been received,
+	A [=Working Group=] could present evidence that a wide review has been received,
```

Comment:
I would not take this change. We use "wide review" as a set phrase, and it is uncountable. This is not a question of how many reviews we get, just of the fact that the activity of reviewing happens.